stdlib/pathlib.pyi

```diff
@@ -147,7 +143,14 @@ class Path(PurePath):
     def open(
         self, mode: str, buffering: int = ..., encoding: str | None = ..., errors: str | None = ..., newline: str | None = ...
     ) -> IO[Any]: ...
-    def owner(self) -> str: ...
+    if sys.platform != "win32":
+        # These methods do "exist" on Windows, but they always raise NotImplementedError,
```

Comment:
Not sure about this, calling code might just catch the NotImplementedError. Let's see what mypy-primer says.

Reply:
Primer seems happy!